subsys/bootloader/Kconfig

```diff
@@ -4,25 +4,13 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-menuconfig SECURE_BOOT
-	bool "Secure Boot"
-	select SECURE_BOOT_CRYPTO
+config SECURE_BOOT
+	bool "Use Secure Boot"
+	depends on !IS_SECURE_BOOTLOADER
```

Comment:
What is the difference between `SECURE_BOOT` and `IS_SECURE_BOOTLOADER`?

Reply:
I will add documentation for this.

The app which enables secure bootloader sets the `SECURE_BOOT` while the bootloader app (ie B0) sets the `IS_SECURE_BOOTLOADER`.